NEWS.md

```diff
@@ -59,7 +59,7 @@
 ### Bug fixes
 
 * Fixed an error where the `RangeFilterState` produced an error when using `bootstrap 4`.
-* Fixed a bug that caused the range slider to omit values selected programmatically through the filter API.
```

Comment:
Is the word "programmatically" removed only to trim the wordlist or is there another reason?